### ISSUE TITLE
Website - Accessibility - Keyboard Users are Not Able to Open Search Results Links

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#75]
A keyboard only or screen reader user is not able to open to what look and like links for the mouse user.

The LINKS seem to be DIVs made to act like LINKS. The links should be created with an A HREF so they users can tab to them with a keyboard and also activate with a keyboard too.

**Comment by jfc3-dol** [comment]
The issue seems to be fixed for the tests I did on the Chrome. I can't test in IE or Firefox because the pages do not display properly.
